app/backend/graphql/task/mutations/createTaskMutationField.ts

```diff
@@ -0,0 +1,29 @@
+import { arg, intArg, mutationField, nullable, stringArg } from 'nexus'
+import { DateScalar } from '../../scalars/date'
+import { Task } from '../task'
+
+export const createTaskMutationField = mutationField('createTaskMutation', {
+  type: Task,
+  description: 'Create a new Task',
+  args: {
+    title: stringArg({ description: 'Title for the task' }),
+    projectId: intArg(),
```

Comment:
I think using `idArg()` here is also better.

Reply:
it is now TaskInput